e2e-tests/tests/440-box-plugin.ts

```diff
@@ -100,13 +101,13 @@ Scenario(
   }
 )
 
-Scenario('Set box type using keyboard', async ({ I }) => {
+Scenario.skip('Set box type using keyboard', async ({ I }) => {
```

Comment:
Todo: Fix

Reply:
You can also use `Scenario.todo`, if you so desire

Reply:
Thanks, fixed the test